easybuild/easyconfigs/l/LAMMPS/LAMMPS-29Sep2021-foss-2021b-CUDA-11.4.1-kokkos.eb

```diff
@@ -0,0 +1,141 @@
+functionality.
+"""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+toolchainopts = {'openmp': True, 'usempi': True, 'cstd': 'c++14'}
+
+# 'https://github.com/lammps/lammps/archive/'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = [
+    'stable_%(version)s.tar.gz',
+]
+patches = [
+    'LAMMPS-29Sep2021-fixdoc.patch',
+    'LAMMPS-29Sep2021-force-cudaaware-kokkos.patch',
+]
+checksums = [
+    '2dff656cb21fd9a6d46c818741c99d400cfb1b12102604844663b655fb2f893d',  # stable_29Sep2021.tar.gz
+    'a17aec5cf67ea3542adf5dfe7eaa318cd0cfa5dbdc30960950e4e85df4774a62',  # LAMMPS-29Sep2021-fixdoc.patch
+    '6a5772cc706562b3de1f5deafdbb60b494f6d3b0c7d2e09623417c0084453be0',  # LAMMPS-29Sep2021-force-cudaaware-kokkos.patch
+]
+
+builddependencies = [
+    ('CMake', '3.21.1'),
+    ('pkg-config', '0.29.2'),
+    ('archspec', '0.1.3'),
+    ('Doxygen', '1.9.1'),
+]
+
+dependencies = [
+    ('CUDA', '11.4.1', '', True),
+    ('NCCL', '2.10.3', local_cudasuffix),
+    ('Python', '3.9.6'),
+    ('libpng', '1.6.37'),
+    ('libjpeg-turbo', '2.0.6'),
+    ('netCDF', '4.8.1'),
+    ('GSL', '2.7'),
+    ('zlib', '1.2.11'),
+    ('gzip', '1.10'),
+    ('cURL', '7.78.0'),
+    ('HDF5', '1.12.1'),
+    ('tbb', '2020.3'),
+    ('PCRE', '8.45'),
+    ('libxml2', '2.9.10'),
+    ('FFmpeg', '4.3.2'),
+    ('Voro++', '0.4.6'),
+    ('kim-api', '2.2.1'),
+    ('Eigen', '3.3.9'),
+    ('PLUMED', '2.7.3'),
+    ('ScaFaCoS', '1.0.1'),
+]
+
+# To use additional custom configuration options, use the 'configopts' easyconfig parameter
+# See docs and lammps easyblock for more information.
+# https://github.com/lammps/lammps/blob/master/cmake/README.md#lammps-configuration-options
+
+# OpenMP-Kokkos build is default in the current easyblock. One can switch to serial backend of Kokkos,
+# which is claimed to be faster in pure MPI calculations
+# configopts  = "-DKokkos_ENABLE_SERIAL=yes "
+
+# packages auto-enabled by easyblock
+# 'GPU'    - if cuda package is present and kokkos is disabled
+# 'KOKKOS' - if kokkos is enabled (by default)
+
+# include the following extra packages into the build
+general_packages = [
+    'ASPHERE',
+    'ATC',
+    'BOCS',
+    'BODY',
+    'CLASS2',
+    'COLLOID',
+    'COLVARS',
+    'COMPRESS',
+    'CORESHELL',
+    'DIFFRACTION',
+    'DIPOLE',
+    'DRUDE',
+    'EFF',
+    'EXTRA-MOLECULE',
+    'EXTRA-PAIR',
+    'FEP',
+    'GRANULAR',
+    'H5MD',
+    'KIM',
+    'KSPACE',
+    'MANIFOLD',
+    'MANYBODY',
+    'MC',
+    'MESSAGE',
+    'MGPT',
+    'MISC',
+    'MOFFF',
+    'MOLECULE',
+    'MOLFILE',
+    'MPIIO',
+    'NETCDF',
+    'OPENMP',
+    'PERI',
+    'PHONON',
+    'PLUMED',
+    'POEMS',
+    'PTM',
+    'PYTHON',
+    'QEQ',
+    'QTB',
+    'REPLICA',
+    'RIGID',
+    'SCAFACOS',
+    'SHOCK',
+    'SMTBQ',
+    'SPH',
+    'SPIN',
+    'SRD',
+    'TALLY',
+    'UEF',
+    'VORONOI',
+]
+
+# required by easyblock
+user_packages = [
+]
+
+# hardware-specific option
+cuda_compute_capabilities = ['8.0']
```

Comment:
This shouldn't be included here, it should be set by the site

Reply:
(but you can comment it out and leave it as a guide)

Reply:
Replacing it with a comment about `cuda_compute_capabilities` would be good. Especially as this can highlight that the LAMMPS easyblock builds for the highest item in the list, as opposed to some other software that produces a fat-binary compatible with all items in that list.